news/2019-09-31-ayim-mca-2018-newspost.md

```diff
@@ -0,0 +1,42 @@
+
+![](IMAGE COMING SOON)
+
+# Mapper's Choice Awards
+
+**The Mapper's Choice Awards forum thread is available in multiple languages, found below:**
+
+[English](https://osu.ppy.sh/community/forums/topics/966003) | [Français](https://osu.ppy.sh/community/forums/topics/966117) | [Deutsch](https://osu.ppy.sh/community/forums/topics/966006) | [한국어](https://osu.ppy.sh/community/forums/topics/966279) | [中文](https://osu.ppy.sh/community/forums/topics/966016) | [日本語](https://osu.ppy.sh/community/forums/topics/965995) | [Русский язык](https://osu.ppy.sh/community/forums/topics/966093) | [Bahasa Indonesia](https://osu.ppy.sh/community/forums/topics/965990) | [Nederlands](https://osu.ppy.sh/community/forums/topics/965967) | [tiếng việt](https://osu.ppy.sh/community/forums/topics/966009)
+
+Mapper's Choice Awards is an event where users eligible can participate in nominating and then voting on the best maps of each of the four game modes as well as storyboards. This event is meant to give perspective as to what the best maps are of 2018 in the eyes of the mapping community! For this event, storyboarding will be regarded as a mode to give the category more focus. Further information about this event can be found on the forum thread.
+
+Each mode will have two **Grand MCA Awards**, one for the best beatmap, and one for the best user. In addition, each mode will have other categories where maps can compete for the top spot such as marathon, spread, and and different music genres. Depending on the mode, there will be other additional categories for voting as well.
+
+Even if you are not able to vote, feel free to follow along with the events! The results will be later on released via livestreams, in which commentators will discuss the results.
+
+## Schedule
+
+As of right now, Mapper's Choice Awards is in the nomination stage, and the rest of the schedule can be seen below. If you're eligible to vote or nominate, be sure to do so while you still can!
+
+- The **Nomination Stage** is held from **September 28th, 00:00 UTC to October 5th, 00:00 UTC** (7 DAYS)
+- The **Voting Stage** is held from **October 7th, 00:00 UTC to October 17th, 18:00 UTC** (11 DAYS)
+- The **Taiko / Catch / Mania Stream** is held on **October 19th**, TIME TBD
+- The **Storyboard / Standard Stream** is held on **October 20th**, TIME TBD
+
+# A Year In Mapping 2018
+
+[A Year In Mapping 2018 forum thread](https://osu.ppy.sh/community/forums/topics/966004)
+
+A Year In Mapping is back again to bring you the achievments and statistics of the mapping community this past year. Each mode is given a section of statistics from information like who ranked the most maps in a mode to the most popular artists and songs. Like 2017, 2018's edition of A Year In Mapping has written commentary on every single person who ranked a map in each mode. If you want to learn more about different members of the mapping community, be sure to check out the thread and given it a read!
```

Comment:
"achievment" --> "achievements"

"Like 2017, 2018's edition of A Year In Mapping has written commentary on every single person who ranked a map in each mode." --> perhaps this could be given further context/clarification as the criteria for a mapper to be included in this year's AYIM are quite different in each modes iirc (for example in osu!standard mappers who only has a GD without a ranked mapset of their own were not included but in osu!catch they were included)

![image](https://user-images.githubusercontent.com/36564236/65820508-45f00980-e254-11e9-996e-372be2838171.png)

Reply:
tried to make it more general